src/main/java/com/yahoo/sketches/hash/MurmurHash3.java

```diff
@@ -76,7 +76,7 @@ private MurmurHash3() {}
     // Get the tail
     long k1 = (rem == 0) ? 0 : key[tail]; //k2 -> 0
     // Mix the tail into the hash and return
-    return hashState.finalMix128(k1, 0, longs * 8); //convert to bytes
+    return hashState.finalMix128(k1, 0, longs * 8L); //convert to bytes
```

Comment:
It is even better to use Long.BYTES here instead of 8L.  Already changed in master.  Similar issues below.